orchestra/communication/tests/test_staffing.py

```diff
@@ -71,6 +73,28 @@ def test_handle_staffing_response_is_available(self):
         self.assertTrue(response.is_winner)
         self.assertEqual(StaffingResponse.objects.all().count(), old_count + 1)
 
+        task_assignment = (
```

Comment:
sanity-check: are all references to FAILED removed with this PR, aside from the ones in machine tasks/